terraform/aws/modules/module_ecs_service/appautoscaling_schedule.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_appautoscaling_scheduled_action" "ecs_schedule_stop" {
+  count              = var.enable_ecs_schedule ? 1 : 0
+  name               = "${local.resource_prefix}_schedule_stop"
+  service_namespace  = "ecs"
+  resource_id        = "service/${aws_ecs_service.ecs_service.cluster}/${aws_ecs_service.ecs_service.name}"
+  scalable_dimension = "ecs:service:DesiredCount"
```

Comment:
is there any setting for desiredcount to 0?

Reply:
There is no argument for that resource to set desire count.  Changing the min and max capacity affects the desire count to what ever they are set too.